ydb/public/tools/lib/cmds/__init__.py

```diff
@@ -310,6 +311,50 @@ def enable_pqcd(arguments):
     return (getattr(arguments, 'enable_pqcd', False) or os.getenv('YDB_ENABLE_PQCD') == 'true')
 
 
+def merge_two_yaml_configs(data_1, data_2):
+    if not (isinstance(data_1, (dict, list)) or data_1 == None):
+        raise TypeError("data_1 is not a dict, list or None")
+    if not (isinstance(data_2, (dict, list)) or data_2 == None):
+        raise TypeError("data_2 is not a dict, list or None")
+    if isinstance(data_1, dict) and isinstance(data_2, dict):
+        new_dict = {}
+        d2_keys = list(data_2.keys())
+        for d1k in data_1.keys():
+            if d1k in d2_keys:
+                d2_keys.remove(d1k)
+                check_types_for_merge(data_1.get(d1k), data_2.get(d1k))
+                new_dict[d1k] = merge_two_yaml_configs(data_1.get(d1k), data_2.get(d1k))
+            else:
+                new_dict[d1k] = data_1.get(d1k)
+                        
+        for d2k in d2_keys:
+            new_dict[d2k] = data_2.get(d2k)
+
+        return new_dict
+    else:
+        if data_2 == None:
+            return data_1
+        else:
+            return data_2
+
+
+def check_types_for_merge(data_1, data_2):
```

Comment:
it is helper function. It will be better to rename it to private variant (with prefix _).